src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -554,7 +543,7 @@ function GlobeControls(view, target, domElement, viewerDiv, radius, getPickingPo
         } else if (this.camera instanceof THREE.OrthographicCamera) {
             this.camera.zoom = Math.max(this.minZoom, Math.min(this.maxZoom, this.camera.zoom * dollyScale));
             this.camera.updateProjectionMatrix();
-            this.dispatchEvent(this.changeEvent);
+            this._view.notifyChange(true);
```

Comment:
What about using: `this._view.notifyChange(true, this.camera);` to indicate to the update cycle that the change was caused by a camera movement? (same comments applies to FPControls obviously)?